src/components/ShareTrajectoryModal/index.tsx

```diff
@@ -0,0 +1,206 @@
+            timeValue = currentTime;
+        } else if (
+            // if user entered time is greater than last frame time
+            inputAsNumber + displayTimes.roundedTimeStep >=
+            displayTimes.roundedLastFrameTime
+        ) {
+            timeValue =
+                displayTimes.roundedLastFrameTime -
+                displayTimes.roundedTimeStep;
+        } else {
+            timeValue = roundToTimestepPrecision(
+                inputAsNumber,
+                displayTimes.roundedTimeStep
+            );
+        }
+        setLastEnteredNumber(timeValue);
+        setUrl(
+            editUrlParams(
+                window.location.href,
+                timeValue.toString(),
+                URL_PARAM_KEY_TIME
+            )
+        );
+    };
+
+    const copyToClipboard = async (): Promise<void> => {
+        try {
+            await navigator.clipboard.writeText(url);
+        } catch (err) {
+            console.error("Failed to copy text: ", err);
+        }
+    };
+
+    const modalOptions = {
+        localFile: {
+            content: (
+                <>
+                    <h4>{Warn} The current file is stored on your device.</h4>
+                    <div className={styles.bodyText}>
+                        <h5>
+                            To generate a shareable link, please save the file
+                            in the public cloud using Dropbox, Google Drive, or
+                            Amazon S3 and load the model into Simularium via
+                            URL.
+                            <a
+                                href="https://simularium.allencell.org/tutorial#share-a-link"
+                                target="_blank"
+                                rel="noreferrer"
+                            >
+                                {" "}
+                                Learn more{" "}
+                            </a>
+                        </h5>
+                    </div>
+                </>
+            ),
+            footer: (
+                <Button
+                    className={styles.okButton}
+                    type="default"
+                    onClick={closeModal}
+                >
+                    Ok
+                </Button>
+            ),
+        },
+        networkedFile: {
+            content: (
+                <>
+                    <div>
+                        <Input
+                            className={styles.urlInput}
+                            value={url}
+                            disabled
+                        />
+                        <Button type="text" onClick={copyToClipboard}>
+                            Copy {Link}
+                        </Button>
+                    </div>
+                    <div className={styles.timeInputContainer}>
+                        {" "}
+                        <Checkbox onChange={handleAllowUserInput}></Checkbox>
+                        <p>Start at</p>
```

Comment:
![image](https://github.com/simularium/simularium-website/assets/30200665/a1b10fa7-2646-4cbe-a4ce-f8a234633bf5)
The "Start at" text looks misaligned on my side?

Reply:
Ack, yes, you're right. Throwing this CSS rule on makes it look ok, but I want a better way. I thought align-items: center was handling it.

.start-time-text {
    padding-top: 8px;
}

Reply:
Tweaked this a bit too, it looks like you could also change the `p` element for "Start at" to `div` and it will line up correctly, the same way the total time label on the right does it. But I'm not sure what other consequences that has for things like accessibility...